### DISCUSSION TITLE
Home Assistant RaspberryMatic CCU läst sich nicht installieren

### DISCUSSION THREAD

**Discussion by Nickman58** [OpenCCU/OpenCCU#2330] [🇩🇪 Fragen&Antworten] [answered]
Hallo,
leider läst sich die RaspberryMatic CCU aus dem Addon Store in Home assistant nicht installieren. Bekomme immer folgende msg.

Can't install ghcr.io/jens-maus/raspberrymatic:3.69.7.20230506: 404 Client Error for http+docker://localhost/v1.42/images/ghcr.io/jens-maus/raspberrymatic:3.69.7.20230506/json: Not Found ("No such image: ghcr.io/jens-maus/raspberrymatic:3.69.7.20230506")

Ich habe es immer wieder versucht. Ohne Erfolg.  Gibt es evtl. eine andere Möglichkeit das Addon zu installieren. Wenn ja, biite ich um eine kleine Anleitung. 
Vielen Dank vorab
Thomas

**Comment by jens-maus** [comment, marked as answer]
Dies kommt immer mal wieder gelegentlich vor und bedeutet nur das entweder gerade GitHub ein Problem hat oder deine IP wegen zuvieler anfragen gerade mitunter geblockt wird. Einfach ein bisschen Geduld haben, es später (in ein paar Minuten / Stunden) nochmal probieren und sicherstellen das das System problemlosen Zugriff ins Internet bzw. zu GitHub hat. Und auch kontrollieren das du genug restlichen Speicherplatz auf dem HA host hast.

**Reply by Nickman58** [reply]
Danke für die schnelle Antwort. Ich versuche es schone seit Tagen. Werde es weiterhin versuchen. Speicherplatz ist genügend vorhanden.

**Reply by Nickman58** [reply]
Hallo, sorry das ich mich nochmals melde. Trotz aller Bemühungen (Minuten / Stunden) ist es mir nach wie vor nicht möglich das Addon zum laufen zu kriegen. Der Zugriff ins Internet (100MBit Anschluss) als auch der Speicherplatz (SD Karte mit 32Gbyie) sind vorhanden.
Die Hardware ist ein Raspberry Pi 3B+. Gibt es irgendiwe eine andere Möglichkeit das Addon zu installieren. 
Vor ein paar Wochen hahe ich ein anderes Addon (Cloudflare) ohne Probleme installieren können
Oder mache ich irgendetwas falsch?

**Reply by dudebimbim** [reply]
Hallo, ich habe seit einem Monat auf meinem Raspberry Pi 2 das gleiche Problem, aber nicht nur bei RaspberryMatic CCU sondern auch bei ESPHome. Im Home Assistant Community Forum [https://community.home-assistant.io/t/unable-to-update-to-latest-esphome/541777/19](url) wird vermutet, dass Home Assistant beim Raspberry Pi 2 auf den ARMv6 Zweig verweist, obwohl Rasbperry P2 auf ARMv7 basiert. Leider kenn ich mich gar nicht mit der Materie aus. Könnte das eine Ursache sein?

**Comment by Baxxy13** [comment]
Ich habe das eben mal schnell mit einem Pi4B durchgetestet.
Mit dem 32bit HA-OS bekomme ich die gleiche Fehlermeldung, mit dem 64bit HA-OS lief die Installation des RaspberrryMatic-AddOns problemlos durch.

Das ist zwar für User die auf 32bit setzen müssen (Pi2B / Pi3B) keine Hilfe, zeigt aber zumindest das es mit der Systemarchitektur zusammenhängt.
